src/containerapp/azext_containerapp/commands.py

```diff
@@ -225,6 +225,12 @@ def load_command_table(self, args):
         g.custom_show_command('show', 'show_admin_for_spring')
         g.custom_command('delete', 'delete_admin_for_spring', confirmation=True, supports_no_wait=True)
 
+    with self.command_group('containerapp env java-component gateway-for-spring', is_preview=True) as g:
+        g.custom_command('create', 'create_gateway_for_spring', supports_no_wait=True)
+        g.custom_command('update', 'update_gateway_for_spring', supports_no_wait=True)
+        g.custom_show_command('show', 'show_gateway_for_spring')
```

Comment:
what about list?

Reply:
The list command is consistent with other Java components at the Java component level.